### ISSUE TITLE
why is -gmt 8 missing from list.

### ISSUE THREAD

**Issue by hittesh** [ndom91/react-timezone-select#16]
hello Team(react-timezone-select),

I am facing a small issue I don't know why it is missing from the options list. I have installed the latest version **(0.9.9)** of your react-timezone-select package. I need (-8 GMT) in my list but it's missing. why is it happening.? Can you please help me?

<img width="1299" alt="Screenshot 2021-01-13 at 4 16 49 PM" src="https://user-images.githubusercontent.com/47388032/104442234-ca39dd80-55ba-11eb-8d75-a43aabae11ed.png">
<select class="modelCountryDropDown" name="rcrs-country" style="margin-left: 9px; color: rgb(73, 80, 87); height: 33px;"><option value="">Select Country</option><option value="AF">Afghanistan</option><option value="AX">Åland Islands</option><option value="AL">Albania</option><option value="DZ">Algeria</option><option value="AS">American Samoa</option><option value="AD">Andorra</option><option value="AO">Angola</option><option value="AI">Anguilla</option><option value="AQ">Antarctica</option><option value="AG">Antigua and Barbuda</option><option value="AR">Argentina</option><option value="AM">Armenia</option><option value="AW">Aruba</option><option value="AU">Australia</option><option value="AT">Austria</option><option value="AZ">Azerbaijan</option><option value="BS">Bahamas</option><option value="BH">Bahrain</option><option value="BD">Bangladesh</option><option value="BB">Barbados</option><option value="BY">Belarus</option><option value="BE">Belgium</option><option value="BZ">Belize</option><option value="BJ">Benin</option><option value="BM">Bermuda</option><option value="BT">Bhutan</option><option value="BO">Bolivia</option><option value="BQ">Bonaire, Sint Eustatius and Saba</option><option value="BA">Bosnia and Herzegovina</option><option value="BW">Botswana</option><option value="BV">Bouvet Island</option><option value="BR">Brazil</option><option value="IO">British Indian Ocean Territory</option><option value="BN">Brunei Darussalam</option><option value="BG">Bulgaria</option><option value="BF">Burkina Faso</option><option value="BI">Burundi</option><option value="KH">Cambodia</option><option value="CM">Cameroon</option><option value="CA">Canada</option><option value="CV">Cape Verde</option><option value="KY">Cayman Islands</option><option value="CF">Central African Republic</option><option value="TD">Chad</option><option value="CL">Chile</option><option value="CN">China</option><option value="CX">Christmas Island</option><option value="CC">Cocos (Keeling) Islands</option><option value="CO">Colombia</option><option value="KM">Comoros</option><option value="CG">Congo, Republic of the (Brazzaville)</option><option value="CD">Congo, the Democratic Republic of the (Kinshasa)</option><option value="CK">Cook Islands</option><option value="CR">Costa Rica</option><option value="CI">Côte d'Ivoire, Republic of</option><option value="HR">Croatia</option><option value="CU">Cuba</option><option value="CW">Curaçao</option><option value="CY">Cyprus</option><option value="CZ">Czech Republic</option><option value="DK">Denmark</option><option value="DJ">Djibouti</option><option value="DM">Dominica</option><option value="DO">Dominican Republic</option><option value="EC">Ecuador</option><option value="EG">Egypt</option><option value="SV">El Salvador</option><option value="GQ">Equatorial Guinea</option><option value="ER">Eritrea</option><option value="EE">Estonia</option><option value="ET">Ethiopia</option><option value="FK">Falkland Islands (Islas Malvinas)</option><option value="FO">Faroe Islands</option><option value="FJ">Fiji</option><option value="FI">Finland</option><option value="FR">France</option><option value="GF">French Guiana</option><option value="PF">French Polynesia</option><option value="TF">French Southern and Antarctic Lands</option><option value="GA">Gabon</option><option value="GM">Gambia, The</option><option value="GE">Georgia</option><option value="DE">Germany</option><option value="GH">Ghana</option><option value="GI">Gibraltar</option><option value="GR">Greece</option><option value="GL">Greenland</option><option value="GD">Grenada</option><option value="GP">Guadeloupe</option><option value="GU">Guam</option><option value="GT">Guatemala</option><option value="GG">Guernsey</option><option value="GN">Guinea</option><option value="GW">Guinea-Bissau</option><option value="GY">Guyana</option><option value="HT">Haiti</option><option value="HM">Heard Island and McDonald Islands</option><option value="VA">Holy See (Vatican City)</option><option value="HN">Honduras</option><option value="HK">Hong Kong</option><option value="HU">Hungary</option><option value="IS">Iceland</option><option value="IN">India</option><option value="ID">Indonesia</option><option value="IR">Iran, Islamic Republic of</option><option value="IQ">Iraq</option><option value="IE">Ireland</option><option value="IM">Isle of Man</option><option value="IL">Israel</option><option value="IT">Italy</option><option value="JM">Jamaica</option><option value="JP">Japan</option><option value="JE">Jersey</option><option value="JO">Jordan</option><option value="KZ">Kazakhstan</option><option value="KE">Kenya</option><option value="KI">Kiribati</option><option value="KP">Korea, Democratic People's Republic of</option><option value="KR">Korea, Republic of</option><option value="KW">Kuwait</option><option value="KG">Kyrgyzstan</option><option value="LA">Laos</option><option value="LV">Latvia</option><option value="LB">Lebanon</option><option value="LS">Lesotho</option><option value="LR">Liberia</option><option value="LY">Libya</option><option value="LI">Liechtenstein</option><option value="LT">Lithuania</option><option value="LU">Luxembourg</option><option value="MO">Macao</option><option value="MK">Macedonia, Republic of</option><option value="MG">Madagascar</option><option value="MW">Malawi</option><option value="MY">Malaysia</option><option value="MV">Maldives</option><option value="ML">Mali</option><option value="MT">Malta</option><option value="MH">Marshall Islands</option><option value="MQ">Martinique</option><option value="MR">Mauritania</option><option value="MU">Mauritius</option><option value="YT">Mayotte</option><option value="MX">Mexico</option><option value="FM">Micronesia, Federated States of</option><option value="MD">Moldova</option><option value="MC">Monaco</option><option value="MN">Mongolia</option><option value="ME">Montenegro</option><option value="MS">Montserrat</option><option value="MA">Morocco</option><option value="MZ">Mozambique</option><option value="MM">Myanmar</option><option value="NA">Namibia</option><option value="NR">Nauru</option><option value="NP">Nepal</option><option value="NL">Netherlands</option><option value="NC">New Caledonia</option><option value="NZ">New Zealand</option><option value="NI">Nicaragua</option><option value="NE">Niger</option><option value="NG">Nigeria</option><option value="NU">Niue</option><option value="NF">Norfolk Island</option><option value="MP">Northern Mariana Islands</option><option value="NO">Norway</option><option value="OM">Oman</option><option value="PK">Pakistan</option><option value="PW">Palau</option><option value="PS">Palestine, State of</option><option value="PA">Panama</option><option value="PG">Papua New Guinea</option><option value="PY">Paraguay</option><option value="PE">Peru</option><option value="PH">Philippines</option><option value="PN">Pitcairn</option><option value="PL">Poland</option><option value="PT">Portugal</option><option value="PR">Puerto Rico</option><option value="QA">Qatar</option><option value="RE">Réunion</option><option value="RO">Romania</option><option value="RU">Russian Federation</option><option value="RW">Rwanda</option><option value="BL">Saint Barthélemy</option><option value="SH">Saint Helena, Ascension and Tristan da Cunha</option><option value="KN">Saint Kitts and Nevis</option><option value="LC">Saint Lucia</option><option value="MF">Saint Martin</option><option value="PM">Saint Pierre and Miquelon</option><option value="VC">Saint Vincent and the Grenadines</option><option value="WS">Samoa</option><option value="SM">San Marino</option><option value="ST">Sao Tome and Principe</option><option value="SA">Saudi Arabia</option><option value="SN">Senegal</option><option value="RS">Serbia</option><option value="SC">Seychelles</option><option value="SL">Sierra Leone</option><option value="SG">Singapore</option><option value="SX">Sint Maarten (Dutch part)</option><option value="SK">Slovakia</option><option value="SI">Slovenia</option><option value="SB">Solomon Islands</option><option value="SO">Somalia</option><option value="ZA">South Africa</option><option value="GS">South Georgia and South Sandwich Islands</option><option value="SS">South Sudan</option><option value="ES">Spain</option><option value="LK">Sri Lanka</option><option value="SD">Sudan</option><option value="SR">Suriname</option><option value="SZ">Swaziland</option><option value="SE">Sweden</option><option value="CH">Switzerland</option><option value="SY">Syrian Arab Republic</option><option value="TW">Taiwan</option><option value="TJ">Tajikistan</option><option value="TZ">Tanzania, United Republic of</option><option value="TH">Thailand</option><option value="TL">Timor-Leste</option><option value="TG">Togo</option><option value="TK">Tokelau</option><option value="TO">Tonga</option><option value="TT">Trinidad and Tobago</option><option value="TN">Tunisia</option><option value="TR">Turkey</option><option value="TM">Turkmenistan</option><option value="TC">Turks and Caicos Islands</option><option value="TV">Tuvalu</option><option value="UG">Uganda</option><option value="UA">Ukraine</option><option value="AE">United Arab Emirates</option><option value="GB">United Kingdom</option><option value="US">United States</option><option value="UM">United States Minor Outlying Islands</option><option value="UY">Uruguay</option><option value="UZ">Uzbekistan</option><option value="VU">Vanuatu</option><option value="VE">Venezuela, Bolivarian Republic of</option><option value="VN">Vietnam</option><option value="VG">Virgin Islands, British</option><option value="VI">Virgin Islands, U.S.</option><option value="WF">Wallis and Futuna</option><option value="EH">Western Sahara</option><option value="YE">Yemen</option><option value="ZM">Zambia</option><option value="ZW">Zimbabwe</option></select>

**Comment by ndom91** [comment]
@hittesh I'm having trouble recreating your issue. The default dropdown contains a `GMT-8:00` value for Tijuana / U.S. Pacific Time.

![image](https://user-images.githubusercontent.com/7415984/104507530-33cfe100-55e7-11eb-9b02-f088bfc902ac.png)

How are you reaching a state where `GMT-8:00` is missing?

**Comment by hittesh** [comment]
hello Team(react-timezone-select),

Please check both screenshots, its not missing its actually shift its -8gmt to -7gmt. why is it happening? 

![image (13) (2)](https://user-images.githubusercontent.com/47388032/104561392-57d60580-566d-11eb-81c5-6afd4a343ef3.png)
<img width="1392" alt="Screenshot 2021-01-14 at 1 35 50 PM" src="https://user-images.githubusercontent.com/47388032/104561506-8522b380-566d-11eb-8bd7-5e0cc54d29b5.png">

**Comment by ndom91** [comment]
That Tijuana/Pacific Time zone option is based off the timezone 'America/Tijuana'. When looking it up, it seems it alternates between -7 and -8 depending on daylight savings time.

![image](https://user-images.githubusercontent.com/7415984/104563795-68c14f80-564a-11eb-9e9a-cd5065490564.png)

**Comment by hittesh** [comment]
I set up 2 apps on the same server one is for development purposes the second one is for production. in both apps pacific time is different. 
 please watch the attached video
https://drive.google.com/file/d/13Srfu36LFPmBSypH7UYpjG5yc_WU09KO/view

**Comment by ndom91** [comment]
Hmm okay. And they are hosted on the same server? 

The server(s) having different timezones is the next first thing that comes to mind.

**Comment by hittesh** [comment]
yes, both apps are hosted on a different server.

**Comment by ndom91** [comment]
Oh so they are on separate servers? Type in `date` on both and copy and paste the output here for me please.

i.e.
![image](https://user-images.githubusercontent.com/7415984/104607151-cbcdd900-5680-11eb-825f-b5248aaf5e66.png)

**Comment by hittesh** [comment]
Please check 
production server(india)
<img width="369" alt="Untitled" src="https://user-images.githubusercontent.com/47388032/104700445-2aa35900-573a-11eb-8c7d-9310c20694f2.png">
live server(us)

<img width="542" alt="Screenshot 2021-01-15 at 2 01 57 PM" src="https://user-images.githubusercontent.com/47388032/104700510-4444a080-573a-11eb-90db-3e13e9d641c4.png">

**Comment by ndom91** [comment]
I think because IST is at an odd interval +5:30 and also doesn't seem like it respects DST, that server is showing you different timezone calculations - I'd bet america/tijuana isn't the only one thats a bit off.

Best practice if you're developing globally is to just set all servers to UTC and calculate offsets for clients locally based on their browser time or whatever the case may be in your application.

**Comment by ndom91** [comment]
@hittesh so the problem is definitely due to the difference in system timezones of your two servers. How to deal with that is up to you, I've made a suggestion in my previous comment.

We just released a new version of this library though (`v0.9.11`) which allows you to add your own timezone choices or completely replace my list of timezone choices with your own - maybe that can fulfill your need as well.

I will close this issue now - if there are further questions, feel free to make a new issue or reopen this one.
